### PR TITLE
PLG-303: Adapt the styles for the Options grid

### PR DESCRIPTION
**Pull request by BryanS29-akeneo**
**Description (for Contributor and Core Developer)**

image

image

image

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

**Review by LaurentPetard — APPROVED** [review]

File: src/Akeneo/Pim/Structure/Bundle/Resources/public/js/attribute-option/components/List.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import {AttributeOption} from '../model';
 import {useAttributeContext} from '../contexts';
```
Comment (rouliane) [inline src/Akeneo/Pim/Structure/Bundle/Resources/public/js/attribute-option/components/List.tsx:5]: This file needs to be removed ? Apparently you replaced the List.tsx by AttributeOptionTable.tsx

File: src/Akeneo/Pim/Structure/Bundle/Resources/public/js/attribute-option/components/AttributeOptionTable.tsx [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    setShowNewOptionPlaceholder(false);
+    if (attributeOptions !== null && attributeOptions.length > 0) {
+      selectAttributeOption(attributeOptions[0].id);
+    }
+  };
+
+  const reorderAttributeOptions = (newIndices: number[]) => {
+    setSortedAttributeOptions((rows: AttributeOption[]): AttributeOption[] => {
+      const newSortedAttributeOptions = newIndices.map(index => rows[index]);
+      if (typeof newSortedAttributeOptions[newSortedAttributeOptions.length - 1] === 'undefined') {
```
Comment (rouliane) [inline src/Akeneo/Pim/Structure/Bundle/Resources/public/js/attribute-option/components/AttributeOptionTable.tsx:75]: You shouldn't have a last "undefined" item in the list, I don't understand

File: src/Akeneo/Pim/Structure/Bundle/Resources/public/js/attribute-option/components/NewOptionPlaceholder.tsx [hunk 3]
```diff
@@ -1,35 +1,65 @@
...
+          level="tertiary"
+        />
+      </TableActionCell>
+    </TableRow>
   );
 };
 
+const TableRow = styled(Table.Row)`
+  td:first-child {
+    color: #f0f1f3;
```
Comment (rouliane) [inline src/Akeneo/Pim/Structure/Bundle/Resources/public/js/attribute-option/components/NewOptionPlaceholder.tsx:46]: We avoid using hardcoded colors, you have to use the DSM for this. You can use a function called `getColor` that is in the DSM

File: src/Akeneo/Pim/Structure/Bundle/Resources/public/js/attribute-option/components/AttributeOptionTable.tsx [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+const HandleContainer = styled.div`
+  cursor: grab;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const TableRow = styled(Table.Row)<{isDraggable: boolean}>`
+  td:first-child {
+    color: ${({isDraggable}) => (isDraggable ? 'grey' : '#f0f1f3')};
```
Comment (rouliane) [inline src/Akeneo/Pim/Structure/Bundle/Resources/public/js/attribute-option/components/AttributeOptionTable.tsx:178]: We avoid using hardcoded colors, you have to use the DSM for this. You can use a function called getColor that is in the DSM

**Review by rouliane — APPROVED** [review]

**Pull request merged**